### DISCUSSION TITLE
Has anyone used this library with tailwind?

### DISCUSSION THREAD

**Discussion by Soybean44** [pelme/htpy#19] [Q&A] [answered]
I have been loving this library, but my only gripe is that tailwind might not be the easiest thing to use with it. If tailwind doesnt work, are there any alternatives that are being worked on? I should state I'm not the most familiar with how to use tailwind in general so I might just be blind

**Comment by Soybean44** [comment, marked as answer]
Did some messing around, turns out its just plug and play with tailwind, just make sure your tailwind config is set to look through python files and youre golden. also make sure to keep recompiling the tailwind css file

**Comment by pelme** [comment]
Hi @Soybean44 , I am happy to hear that you find htpy useful! Thanks for the heads up about Tailwind! :)
